api/embed.go

```diff
@@ -3,12 +3,22 @@ package api
 import (
 	_ "embed"
 	"fmt"
+	"hash/crc64"
 	"net/http"
 )
 
 //go:embed openapi.gen.json
 var embeddedJSONSpec []byte
 
+// ETagJSONSpec contains sha sum of the SPEC json.
+var ETagJSONSpec string
+
+func init() {
+	hash := crc64.New(crc64.MakeTable(crc64.ECMA))
+	hash.Write(embeddedJSONSpec)
+	ETagJSONSpec = fmt.Sprintf("%x", hash.Sum64())
+}
+
```

Comment:
Calculates CRC64 hash of the whole JSON spec file during startup and that is returned to the browser.